git/packet_writer.go

```diff
@@ -0,0 +1,124 @@
+package git
+
+import (
+	"io"
+
+	"github.com/github/git-lfs/tools"
+)
+
+type PacketWriter struct {
+	// buf is the internal buffer of bytes used to store data given in calls
+	// to Write that don't warrant a protocol write.
```

Comment:
"...don't warrant a protocol write." --> might be the late hour or my non-native english ... but I don't get this sentence 🤔